nop/docs.go

```diff
@@ -0,0 +1,4 @@
+/*
+Package nop contains an implementation of the `gokv.Store` interface that does nothing.
+*/
+package nop
```

Comment:
WDYT about `noop`? Could it be more common and then more obvious for people what it means?

Reply:
I will work on it

Reply:
I can also do the change if you don't have time. If that's okay for you?

Reply:
ok I just had time to perform this modification with attention